pkg/controllers/options/resources.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package options
```

Comment:
Nit: Not sure if we need this all in a separate file. Can just be a part of the other file.

Reply:
it could, I was thinking that we could reuse this file in the future too. So that when there's a need to implement something like this again, we don't have to look for places where we put an "enum" or resources and reuse, but reuse this file as a whole.